styles/variables.js

```diff
@@ -5,6 +5,9 @@ export default {
   gap: {
     small: 8,
     big: 16,
+    bigger: 20,
```

Comment:
you know what will happen, if you start something like that. at least `xbig` and `xxbig` could be avoided. by multiplying `big` within the components.

Reply:
I don't like the idea of multiplying variables in the components since this constant is on the designs. Ergo, it will be used more and more in the future.

Xbig is already used in the loading indictor which imho should be changed to xbig.

I choose to name 20px "bigger" since it's not a multiplication, but it is required in provided designs.

Reply:
keep it like it is :-)